config/chime_science_run_gpu.yaml

```diff
@@ -535,6 +535,29 @@ gpu:
       rfi_output_buf: gpu_rfi_output_buffer_3
       rfi_mask_output_buf: gpu_rfi_mask_output_buffer_3
 
+
+rfi_update_metadata:
+  update_metadata_0:
+    kotekan_stage: rfiUpdateMetadata
+    rfi_mask_buf: gpu_rfi_mask_output_buffer_0
+    lost_samples_buf: lost_samples_buffer
+    gpu_correlation_buf: gpu_n2_output_buffer_0
```

Comment:
Maybe add a comment here in the config to explain why this is used as an output of this as well as `hsaProcess` stage? It had confused me.